contracts/test/MapleGlobals.t.sol

```diff
@@ -1,273 +1,439 @@
-        assertTrue(     gov.try_setPriceOracle(WETH, address(wethOracle)));
-        assertTrue(     gov.try_setPriceOracle(WBTC, address(wbtcOracle)));
-        assertEq(globals.oracleFor(WETH),            address(wethOracle));
-        assertEq(globals.oracleFor(WBTC),            address(wbtcOracle));
-
-        assertTrue(globals.getLatestPrice(WETH) != 0);  // Shows real WETH value from Chainlink
-        assertTrue(globals.getLatestPrice(WBTC) != 0);  // Shows real WBTC value from Chainlink
-
-        // setMaxSwapSlippage()
-        assertEq(   globals.maxSwapSlippage(),      1_000);
-        assertTrue(!fakeGov.try_setMaxSwapSlippage(10_000));
-        assertTrue(    !gov.try_setMaxSwapSlippage(10_001));  // 100.01% is outside of bounds
-        assertTrue(     gov.try_setMaxSwapSlippage(10_000));  // 100% is upper bound
-        assertEq(   globals.maxSwapSlippage(),     10_000);
-
-        // setValidBalancerPool()
         assertTrue(!globals.isValidBalancerPool(address(1)));
-        assertTrue(!fakeGov.try_setValidBalancerPool(address(1), true));
-        assertTrue(     gov.try_setValidBalancerPool(address(1), true));
+
+        assertTrue(!fakeGov.try_mapleGlobals_setValidBalancerPool(address(globals), address(1), true));
+        assertTrue( realGov.try_mapleGlobals_setValidBalancerPool(address(globals), address(1), true));
         assertTrue( globals.isValidBalancerPool(address(1)));
 
-        // setMinLoanEquity
-        assertEq(   globals.minLoanEquity(),      2_000);
-        assertTrue(!fakeGov.try_setMinLoanEquity(10_000));
-        assertTrue(    !gov.try_setMinLoanEquity(10_001));  // 100.01% is outside of bounds
-        assertTrue(     gov.try_setMinLoanEquity(10_000));  // 99.99 %
-        assertEq(   globals.minLoanEquity(),     10_000);   // 100% is upper bound
+        assertTrue(!fakeGov.try_mapleGlobals_setValidBalancerPool(address(globals), address(1), false));
+        assertTrue( realGov.try_mapleGlobals_setValidBalancerPool(address(globals), address(1), false));
+        assertTrue(!globals.isValidBalancerPool(address(1)));
+    }
+    
+    function test_setProtocolPause() public {
+        Governor     realGov         = new Governor();
+        GlobalAdmin  realGlobalAdmin = new GlobalAdmin();
+        GlobalAdmin  fakeGlobalAdmin = new GlobalAdmin();
+        MapleGlobals globals         = new MapleGlobals(address(realGov), address(1), address(realGlobalAdmin));
+        
+        assertTrue(!globals.protocolPaused());
+
+        assertTrue(!fakeGlobalAdmin.try_mapleGlobals_setProtocolPause(address(globals), true));
+        assertTrue( realGlobalAdmin.try_mapleGlobals_setProtocolPause(address(globals), true));
+        assertTrue( globals.protocolPaused());
+
+        assertTrue(!fakeGlobalAdmin.try_mapleGlobals_setProtocolPause(address(globals), false));
+        assertTrue( realGlobalAdmin.try_mapleGlobals_setProtocolPause(address(globals), false));
+        assertTrue(!globals.protocolPaused());
```

Comment:
`makeGlobalsAndGovs` not used again. Perhaps throw this in the other contract and `MapleGlobalsGeneralTests`.